.github/workflows/pr-golangci-lint.yml

```diff
@@ -1,4 +1,4 @@
-name: golangci-lint
```

Comment:
While we're at it. Let's please use `.yaml` consistently (there are probably references to the action file names somewhere)

(please also update jobs.md)